webserver/webserver/src/test/java/io/helidon/webserver/Status204Test.java

```diff
@@ -59,12 +52,18 @@ public void stopServer() throws Exception {
     }
 
     @Test
-    public void callPutAndGet() {
-        WebTarget target = ClientBuilder.newClient()
-                                        .target("http://localhost:" + server.port());
-        Response response = target.request().put(Entity.entity("test call", MediaType.TEXT_PLAIN));
-        assertThat(response.getStatus(), is(204));
-        String s = target.request().get(String.class);
-        assertThat(s, is("test"));
+    public void callPutAndGet() throws Exception {
+        WebClient webClient = WebClient.builder()
+                .baseUri("http://localhost:" + server.port())
```

Comment:
use flow all the way, when client is not reused.

Reply:
I can't do that since I need WebClient instance to make another call in the flow.